patches/components-sync_bookmarks-bookmark_remote_updates_handler.cc.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/components/sync_bookmarks/bookmark_remote_updates_handler.cc b/components/sync_bookmarks/bookmark_remote_updates_handler.cc
+index 8b81f03c143f4f036d56d92c62f2c15ded7ec81d..85c6bcaf6d4c14c71b7f17426f415e2d1aa32f87 100644
+--- a/components/sync_bookmarks/bookmark_remote_updates_handler.cc
++++ b/components/sync_bookmarks/bookmark_remote_updates_handler.cc
+@@ -144,7 +144,7 @@ void ApplyRemoteUpdate(
+   // Compute index information before updating the |tracker|.
+   const size_t old_index = size_t{old_parent->GetIndexOf(node)};
+   const size_t new_index =
+-      ComputeChildNodeIndex(new_parent, update_entity.unique_position, tracker);
++      ComputeChildNodeIndex(new_parent, update_entity.unique_position, tracker) BRAVE_APPLY_REMOTE_UPDATE;
```

Comment:
do we have to change this line? Can't we check against the value of `new_index` ? I'm also concerned that we're using a ternary operator with an int. It's better to make a int value check there

Reply:
actually looking at this more closely I think `BRAVE_APPLY_REMOTE_UPDATE` should just const cast `new_index` and change it

Reply:
fixed in 9b84b78a4497e259b5303d3420c81be09d6f1ef9